xsuite/src/cli/newWalletCmd.ts

```diff
@@ -37,19 +44,47 @@ const action = async ({
       return;
     }
   } else {
-    keystore = Keystore.createFile_unsafe(walletPath, password);
+    let data;
+    if (fromPemPath && fromWalletPath)
```

Comment:
if () {}

Same for a few other places.

Ideally, this should be caught by the linter but I didn't succeed to make it catch it correctly. When the `if () ...` holds in one line, I would like no brace, but when it doesn't, I would like one.

Reply:
The eslint rule `"curly": ["error", "multi-line"]` seems to work.

Reply:
Indeed, thanks!